src/odemis/gui/comp/viewport.py

```diff
@@ -1193,6 +1193,7 @@ def __init__(self, *args, **kwargs):
 
         self.canvas.Bind(wx.EVT_MIDDLE_DOWN, self.on_drag_start)
         self.canvas.Bind(wx.EVT_MIDDLE_UP, self.on_drag_end)
+        self.canvas.Bind(wx.EVT_LEAVE_WINDOW, self.on_drag_end)
```

Comment:
Shouldn't it be more like in `DragMixin`?
`self.cnvs.Bind(wx.EVT_KILL_FOCUS, self._on_focus_lost)`

Reply:
Should this one bind too to the drag end function you think?

It seems to work

Reply:
Seems like this breaks the view port from loading?